src/Symfony/Component/Notifier/Bridge/JoliNotif/JoliNotifOptions.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Component\Notifier\Bridge\JoliNotif;
+
+use Symfony\Component\Notifier\Exception\InvalidArgumentException;
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+
+/**
+ * @author Ahmed Ghanem <ahmedghanem7361@gmail.com>
+ */
+final class JoliNotifOptions implements MessageOptionsInterface
+{
+    private array $options = [];
+
+    public function toArray(): array
+    {
+        return $this->options;
+    }
+
+    public function getRecipientId(): ?string
+    {
+        return null;
+    }
+
+    /**
+     * @return $this
+     */
+    public function setIconPath(string $iconPath): self
```

Comment:
`: static` to be inline with `@return $this`

Reply:
Done